File: AD-HOC/1428/main.c
#include <stdio.h>

int main() {
    int t, n, m;
    scanf("%d", &t);
    for (int times = 0; times < t; times++) {
        scanf("%d %d", &n, &m);
        int total = (n / 3) * (m / 3);
        printf("%d\n", total);
    }
    return 0;
}

File: AD-HOC/1428/main.dart
import 'dart:io';

void main() {
  int t = int.parse(stdin.readLineSync()!);
  for (int times = 0; times < t; times++) {
    var [n, m] = stdin.readLineSync()!.split(' ').map(int.parse).toList();
    int total = (n ~/ 3) * (m ~/ 3);
    print(total);
  }
}
